### PR TITLE
from fysnet: PID code for ATTINY2313

### PR DESCRIPTION
**Pull request by fysnet**
I just learned about you from www.hackaday.com.  I have always wanted my own PID code for a small project that I work on.

I am requesting PID number 2222, and hope that I have done the "fork" as you have requested.

The link(s) to this project is within the index.md files you requested.

Thank you,
Ben
fys@fysnet.net

### CODE REVIEW

**Comment by Arachnid** [comment]
Thanks for the pull request! However, your source code link is 404. Also, PID 0x2222 has been claimed; please claim a different PID.

**Comment by Arachnid** [comment]
You need to remove the original PID file from your commit; you also need to link to a valid source repository with the 'source' line; the one you're linking to is 404.

**Comment by fysnet** [comment]
I believe I have done so now.  New PID is A55A.  I will see about removing /2222.

Thank you.

**Comment by Arachnid** [comment]
Thanks. You still need to add a LICENSE file to your source repository. Once you have, we're good to go.

**Comment by fysnet** [comment]
Okay, added the GPLv2 license tag.  Thank you for your help.  Please be sure to let me know if I must do anything else.  Ben.

**Comment by Arachnid** [comment]
I meant to say that you need to add the LICENSE to the repository where you keep your source code. It's all very well to say that your code is licensed under GPLv2, but currently it's not.

You also need to delete the leading blank lines from the PID file, please.

**Comment by fysnet** [comment]
I am truly sorry for the trouble.  I am trying to do this and work at the same time.  I would wait until I have time, but my first choice of PID was taken 5 minutes before I first submitted :-)  Therefore, I am trying to rush this along before my next choice it taken.  So, again, sorry for the trouble.

I hope that it is now what you need.

Thank you.

File: 1209/A55A/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: pid
+title: USB: The Universal Serial Bus: ATTINY2313
+owner: Forever Young Software
```
Comment (Arachnid) [inline 1209/A55A/index.md]: This needs to be the name of the directory your org file is in, in this case 'fysnet'.

**Comment by fysnet** [comment]
The 'owner:' line?  If so, it is now done.

**Comment by fysnet** [comment]
Thank you for doing all of this for us.  It is appreciated by many.  Thanks.

**Pull request merged**